src/app/_components/PageHeader.tsx

```diff
@@ -34,38 +36,35 @@ function Title({ children }: TitleProps) {
   )
 }
 
-const sharedContainerStyle = 'relative aspect-video lg:aspect-auto lg:w-1/2'
-const sharedImageStyle = 'h-full w-full rounded-lg border border-brand-100'
-const imageSizes = buildImageSizeProp({ startSize: '100vw', md: '660px' })
+const aspectRatioStyle = 'aspect-video'
+const borderStyle = 'rounded-lg border border-brand-100'
+const imageSizes = buildImageSizeProp({ startSize: '100vw', lg: '490px' })
```

Comment:
Should this maybe be `lg: '480px'`?

<img width="506" alt="Screenshot 2024-06-21 at 13 43 12" src="https://github.com/FilecoinFoundationWeb/filecoin-foundation/assets/50910606/9682c005-c614-4e02-9a6a-edb0588a6c87">

Reply:
I usually add a few extra pixels to the `sizes` prop so that the image chosen by the browser is slightly bigger than the size of the container to avoid pixelization / too low quality images.